qa/tasks/ceph.py

```diff
@@ -271,10 +272,15 @@ def write_rotate_conf(ctx, daemons):
                         run.Raw('|'),
                         'sudo',
                         'xargs',
+                        '--max-args=1',
+                        '--max-procs=0',
+                        '--verbose',
                         '-0',
                         '--no-run-if-empty',
                         '--',
                         'gzip',
+                        '-5',
```

Comment:
Just curious what kind of difference you observed between `-6` (default) and `-5` for a typical OSD or MDS log?

Reply:
I didn't check. I had experimented with `-9` but it was too slow so I toned it down to `-5`.